File: ReSharper.FSharp/src/FSharp.Psi.Features/src/Fsi/ZoneMarker.fs
namespace JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Fsi

open JetBrains.Application.BuildScript.Application.Zones
open JetBrains.Platform.RdFramework

[<ZoneMarker>]
type ZoneMarker() =
    interface IRequire<IRdFrameworkZone>
    interface IRequire<ISinceClr4HostZone>

File: ReSharper.FSharp/src/FSharp.Psi/src/Resolve/ZoneMarker.cs
using JetBrains.Application.BuildScript.Application.Zones;
using JetBrains.Platform.RdFramework;

namespace JetBrains.ReSharper.Plugins.FSharp.Psi.Resolve
{
  [ZoneMarker]
  public class ZoneMarker : IRequire<IRdFrameworkZone>, IRequire<ISinceClr4HostZone>
  {
  }
}